### PR TITLE
Add managed postgresql page

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Added a new page for Managed PostgreSQL

[Design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=1716-33705&node-type=frame&t=6rKkPmT85JT1F2Qv-0)

## QA
- Go to https://canonical-com-1395.demos.haus/data/postgresql/managed
- Compare with [copy doc](https://docs.google.com/document/d/10YAubAK42OWeaAXY2Ib9OE1q65xyP741R4bnnAA9GCk/edit?tab=t.0)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-15582

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
nice work as usual @steverydz!

couple of small things I noticed:

- pls add `-on-large` to switch the hero into single column mode on medium screens:

image

- pls add 16:9 aspect ratio to this. Vanilla example [here]

(https://vanillaframework.io/docs/examples/patterns/image/container/aspect-ratio/all?theme=light)

image

- we have an extra p-section--shallow here, causing this section to have more spacing at the bottom than the rest - can we please remove (and add u-no-margin--bottom on the ul as well if possible):

image

- pls make these strip deep, for added emphasis:
 
image
image

**Comment by eliman11** [comment]
Thank you @steverydz! A few additional things from me: 
- [ ] Could you please add alt text (PostgreSQL) for the logo in the header?
- [ ] Upon closing the contact us pop-up form, it seems to redirect me to /data/postgresql instead of remaining on the /data/postgresql/managed page
- [ ] Change background colour from white into paper/light grey (F3F3F3)
- [ ] The hr (rulers, separators) above and/or below the CTA sections should span the width of the grid instead of the whole page
Screenshot 2024-10-11 at 15 30 22
Screenshot 2024-10-11 at 15 30 30
- [ ] Unrelated to this page, but the logo size is inconsistent with the [bubble home](https://canonical-com-1395.demos.haus/data/postgresql) - would it be possible to change the sizing in /data/postgresql so it's consistent with this data/postgresql/managed page?

**Comment by steverydz** [comment]
@lyubomir-popov @eliman11 All changes addressed

**Comment by eliman11** [comment]
Looks good to me, thanks Steve! I'm waiting on IS to give me writing permissions so can't change the label, but feel free to add UX+1 from me.

**Comment by lyubomir-popov** [comment]
LGTM, thanks @steverydz

File: templates/data/postgresql/managed.html [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+<div class="u-fixed-width">
+  <hr class="p-rule" />
+</div>
+<section class="p-strip is-deep">
+  <div class="u-fixed-width">
+    <h2><a href="https://ubuntu.com/managed">Learn more about Managed Services&nbsp;&rsaquo;</a></h2>
+  </div>
+</section>
+<section class="p-section">
+  <div class="u-fixed-width">
```
Comment (lizzochek) [inline templates/data/postgresql/managed.html:396]: May as well put the hr into the row to make it cleaner

File: templates/data/postgresql/managed.html [hunk 2]
```diff
@@ -0,0 +1,465 @@
...
+<div class="u-fixed-width">
+  <hr class="p-rule" />
+</div>
+<section class="p-strip is-deep">
+  <div class="u-fixed-width">
+    <h2><a href="/data/postgresql#get-in-touch">Discuss your needs with our team&nbsp;&rsaquo;</a></h2>
+  </div>
+</section>
+<section class="p-section">
+  <div class="u-fixed-width">
```
Comment (lizzochek) [inline templates/data/postgresql/managed.html:448]: Same as for the other hr

File: templates/data/postgresql/managed.html [hunk 3]
```diff
@@ -0,0 +1,465 @@
...
+    <div class="col-6 col-start-large-7">
+      <p><small class="u-text--muted">“PostgreSQL” is a trademark or registered trademark of <a href="https://www.postgres.ca/">PGCA</a>. Other trademarks are property of their respective owners. Canonical’s PostgreSQL offerings are not sponsored, endorsed, or affiliated with <a href="https://www.postgres.ca/">PGCA</a>.</small></p>
+    </div>
+  </div>
+</section>
+
+<script defer src="{{ versioned_static('js/modals.js') }}"></script>
+
+{% with %}
+  {% set returnURL = "https://canonical.com/data/postgresql/managed#success" %}
```
Comment (lizzochek) [inline templates/data/postgresql/managed.html:461]: For c.com links pls remove site name and use only "/data/postgresql/managed#success"

**Review by lizzochek — APPROVED** [review]
Add a couple minor comments, but other than that really well done

**Pull request merged**